### PR TITLE
Mullvad removed support for port forwarding

### PR DESCRIPTION
**Pull request by namazso**
https://mullvad.net/en/blog/2023/5/29/removing-the-support-for-forwarded-ports/

Changes proposed in this PR:

- 

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Comment by namazso** [comment]
I think I broke the IPv6 thing in the outline on the right, but not sure how, or how to fix it

image

File: docs/vpn.md [hunk 1]
```diff
@@ -155,10 +155,6 @@ Mullvad [recommends](https://mullvad.net/en/help/why-wireguard/) the use of Wire
 
 Mullvad supports the future of networking [IPv6](https://en.wikipedia.org/wiki/IPv6). Their network allows you to [access services hosted on IPv6](https://mullvad.net/en/blog/2014/9/15/ipv6-support/) as opposed to other providers who block IPv6 connections.
```
Comment (jonaharagon) [inline docs/vpn.md:157]:
```suggestion

#### :material-alert-outline:{ .pg-orange } Remote Port Forwarding

Mullvad previously supported port forwarding, but removed the option in [May 2023](https://mullvad.net/en/blog/2023/5/29/removing-the-support-for-forwarded-ports/). Missing this feature could negatively impact certain applications, especially peer-to-peer applications like torrent clients.
```

File: docs/vpn.md [hunk 2]
```diff
@@ -155,10 +155,9 @@ Mullvad [recommends](https://mullvad.net/en/help/why-wireguard/) the use of Wire
...
 
-#### :material-check:{ .pg-green } Remote Port Forwarding
-
-Remote [port forwarding](https://en.wikipedia.org/wiki/Port_forwarding) is allowed for people who make one-time payments, but not allowed for accounts with a recurring/subscription-based payment method. This is to prevent Mullvad from being able to identify you based on your port usage and stored subscription information. See [Port forwarding with Mullvad VPN](https://mullvad.net/help/port-forwarding-and-mullvad/) for more information.
+#### :material-alert-outline:{ .pg-orange } Remote Port Forwarding
 
+Mullvad previously supported port forwarding, but removed the option in [May 2023](https://mullvad.net/en/blog/2023/5/29/removing-the-support-for-forwarded-ports/). Missing this feature could negatively impact certain applications, especially peer-to-peer applications like torrent clients.
```
Comment (jonaharagon) [inline docs/vpn.md:160]:
```suggestion
Mullvad previously supported port forwarding, but removed the option in [May 2023](https://mullvad.net/en/blog/2023/5/29/removing-the-support-for-forwarded-ports/). Missing this feature could negatively impact certain applications, especially peer-to-peer applications like torrent clients.

```

**Comment by dngray** [comment]
> I think I broke the IPv6 thing in the outline on the right, but not sure how, or how to fix it

Yeah that doesn't need to be removed from the abbreviation file.

**Review by jonaharagon — APPROVED** [review]
The IPv6 bug is unrelated to this PR, so we can merge this: https://github.com/squidfunk/mkdocs-material/issues/5579

**Review by dngray — APPROVED** [review]

**Pull request merged**